app/server/api/credit_transfer_api.py

```diff
@@ -231,16 +238,24 @@ def post(self, credit_transfer_id):
                 return make_response(jsonify(response_object)), 400
 
             transfer_user_list = []
-            for transfer_account_id in recipient_transfer_accounts_ids:
-                try:
-                    individual_sender_user = None
-                    individual_recipient_user = find_user_with_transfer_account_from_identifiers(
-                        None, None, transfer_account_id)
+            individual_sender_user = None
+
+            if invert_recipient_list:
+                all_accounts_query = TransferAccount.query.filter(TransferAccount.is_ghost != True).filter_by(organisation_id=g.active_organisation.id)
```

Comment:
Note: For this to work with search, this query (the set we're inverting) will have to be replaced with the original search query. This is just getting the skeleton in place.